.github/workflows/reusable_build_packages.yaml

```diff
@@ -166,7 +265,7 @@ jobs:
 
   build-wasm-package:
     if: ${{ inputs.arch == 'x86_64' }}
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Same fix we applied in https://github.com/falcosecurity/libs/pull/2203

Reply:
yep, wouldn't build without it